spec/models/page_spec.rb

```diff
@@ -256,5 +256,14 @@
       expect(Page.new.action_count).to eq 0
     end
   end
+
+  describe 'find by slug' do
+    it 'finds a page by its slug' do
+      expect(Page.find('simple_slug')).to eq simple_page
+    end
+    it 'finds a page by id' do
+      expect(Page.find(simple_page.id)).to eq simple_page
```

Comment:
Nah it doesn't work - but I tested around and it doesn't seem to be an issue - `params[:page_id]` always comes as the numeric id.